johnslist/dbtest/templates/dbtest/job_creation.html

```diff
@@ -167,13 +167,13 @@
 
                         <h1>Job Categories</h1>
                         Select some words that pertain to your job and we will try to find a group that fits your needs.
+                        {% render_field form.categories %}
```

Comment:
Minor: I think we probably need to put some comments or write a detailed documentation on render_field in project documentation since it's not documented well in Django website.